src/screens/discover/DiscoverScreen.tsx

```diff
@@ -87,7 +87,7 @@ export default function DiscoverScreen() {
           contentContainerStyle={isSearchModeEnabled ? { height: '100%' } : {}}
           scrollEnabled={!isSearchModeEnabled}
           bounces={!isSearchModeEnabled}
-          removeClippedSubviews
+          removeClippedSubviewsx
```

Comment:
This is the same as removing it. Do we want to remove it for iOS too? If not just check for android and set it to false

Reply:
ooops looks like i accidentally typed an extra character here, there should be no change